README.md

```diff
@@ -15,17 +15,18 @@ LinuxKit, a toolkit for building custom minimal, immutable Linux distributions.
 
 ## Getting Started
 
-### Build the `moby` tool
+### Build the `moby` and `linuxkit` tools
```

Comment:
Should we add a brief explanation of the difference between the linuxkit and moby tools?  I realize we'll continue to shuffle things around but might be helpful in the intermediate stage